packages/cysync-core/src/dialogs/Inheritance/GoldPlanPurchase/context/types.ts

```diff
@@ -11,6 +11,7 @@ import {
 
 export interface IWalletWithDeleted extends IWallet {
   isDeleted?: boolean;
+  isActive?: boolean;
 }
```

Comment:
The name of this interface is obsolete now, rename accordingly. instead of naming it something `IWalletWithDeletedAndActive` name it something that conveys the purpose, something along the lines of `IWalletForSelection`